source/features/unwrap-unnecessary-dropdowns.tsx

```diff
@@ -36,12 +36,44 @@ async function unwrapNotifications(): Promise<void | false> {
 	button.textContent = `Group by ${button.textContent.toLowerCase()}`;
 }
 
+function unwrapRerunActions(): void {
+	const menu = $('.PageHeader-actions action-menu');
+
+	const dropdown = $('anchored-position', menu);
+	if (!dropdown) {
+		return;
+	}
+
+	const triggerButton = $('focus-group > button', menu);
+
+	// We only need to unwrap the re-run jobs menu
+	if (triggerButton.textContent.trim() !== 'Re-run jobs') {
+		return;
+	}
+
+	const container = menu.parentElement!;
+	container.classList.add('d-flex', 'gap-2');
+
+	const buttons = $$('button.ActionListContent', dropdown);
+	for (const button of buttons) {
```

Comment:
```suggestion
	for (const button of $$('button.ActionListContent', dropdown)) {
```

I'd suggest using the strict version where possible. We want to move away from the loose one eventually

Reply:
See comment

Reply:
😅 I have to wear my glasses.